cmake/process_users_input.cmake

```diff
@@ -31,10 +31,11 @@ option(LIKWID "Link with the likwid marker interface for proxy" OFF)
 option(INTEGRATE_QUANTITIES "Compute cell-averaged integrated velocity and stress components (currently breaks compilation)" OFF)
 option(ADDRESS_SANITIZER_DEBUG "Use address sanitzer in debug mode" OFF)
 
-option(TESTING "Compile unit tests" OFF)
-option(TESTING_GENERATED "Include kernel tests generated by yateto" OFF)
+option(TESTING "Compile unit tests and collect them for execution" OFF)
+option(TESTING_GENERATED "Include kernel unit tests as generated by yateto" OFF)
 option(COVERAGE "Generate targed for code coverage using lcob" OFF)
 set(TESTING_COMMAND "" CACHE STRING "Prefix to the test binary, so that CMake can list the tests")
+option(TESTING_BUILD_ONLY "Only compile the unit tests, but not collect them (only in effect if TESTING=ON)" OFF)
```

Comment:
"but do not"
maybe collect means run here?
(because I would think you need to collect first to be able to compile?)